src/utils/bigNumbers.ts

```diff
@@ -54,3 +52,9 @@ export const unpadLeadingZerosString = (num: string) => {
   }
   return `0x${num}`
 }
+
+export const toHexString = (num: bigint) => {
```

Comment:
We might be able to do this with https://docs.ethers.org/v4/api-utils.html#hex-strings